.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -248,7 +252,10 @@ jobs:
             if [[ $NUM_TEST_SHARDS -ne 2 ]]; then
               export SHARD_NUMBER=0
             fi
-            if [[ -n $GITHUB_HEAD_REF && "$RUN_SMOKE_TESTS_ONLY_ON_PR" == "true" ]]; then
+            if [[ "$TEST_CONFIG" = "force_on_cpu" ]]; then
```

Comment:
Should we just add this to the `win-test.sh` script instead of in the workflow file?